docs/develop/node/compile-and-run-a-node.md

```diff
@@ -0,0 +1,265 @@
+$ git checkout tags/1.19.0 -b mynode
+```
+
+### 2. Compile nearcore binary
+
+In the `nearcore` folder run the commands:
+
+```bash
+$ cargo build --release --package neard --bin neard
+```
+
+This will start the compilation process, it will take some time depending on your machine power (e.g. i9 8-core CPU, 32 GB RAM, SSD it takes approximately 25 minutes)
+
+The binary path is `nearcore/target/release/neard`
+
+### 3. Initialize working directory
+
+In order to work NEAR node requires to have working directory and a couple of configuration files.
+
+- `config.json` with a lot of parameters which responsive for how the node will work
+- `genesis.json` a file with all the data the network had started with. In contains initial accounts, contracts, access keys and other records which represents the initial state of the blockchain
+- `node_key.json` a file which contains public and private key for the node. Also includes optional `account_id` parameter which is required to run validator node (not covered in this doc)
+- `data/` folder in which NEAR node will write the state.
+
+Initially the required working directory needs to be generated. The command to generate it looks like:
+
+```bash
+$ ./target/release/neard --home ~/.near init --chain-id mainnet --download
+```
+
+> You can skip `--home` argument if you are fine with default working directory in `~/.near` or pass different path
+
+This command will create the required directory structure, will generate `config.json` and `node_key.json`, will download `genesis.json` for mainnet network.
+
+
+### 4. Replacing the `config.json`
+
+Generated `config.json` is missing `boot_nodes` parameter (it is empty) so we will need to replace it by full one. The file can be downloaded:
+
+- Download `config.json` https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/mainnet/config.json and replace it in your working dir (`~/.near/config.json`)
+
+Or run the commands:
+
+```bash
+$ rm ~/.near/config.json
+$ wget ~/.near/config.json https://s3-us-west-1.amazonaws.com/build.nearprotocol.com/nearcore-deploy/mainnet/config.json
+```
+
+### 5. Get data backup
+
+The node is ready to be started. However the first thing to do is to sync up with the netowk. It means your node needs to download all the headers and blocks like other nodes in the network have. This process might be speed up drastically by using backups.
+
+Download and unpack the tar to `~/.near` https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/rpc/data.tar
+
+Or run the commands
+
+```bash
+$ wget ~/.near/data.tar https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/rpc/data.tar
+$ tar -xf ~/.near/data.tar
+$ rm ~/.near/data.tar
+```
+
+### 6. Run a node
+
+```bash
+$ ./target/release/neard --home ~/.near run
+```
+
+That's all. The node is running you can see log outputs in your console. It will download a bit of missing data since the backup was done but it shouldn't take much time.
```

Comment:
```suggestion
That's all. The node is running and you can see log outputs in your console. It will download a bit of missing data since the last backup was performed but it shouldn't take much time.
```